main/webapp/modules/core/langs/translation-fr.json

```diff
@@ -0,0 +1,670 @@
+		"from-total": "</b> total",
+		"cluster-descr": "Cet outil vous aide à identifier des groupes de cellules ayant des valeurs différentes mais qui peuvent correspondre à des représentations alternatives de la même valeur. Par exemple, les deux chaînes \"New York\" et \"new york\" n’ont qu’une différence de casse et font très certainement référence à la même ville. \"Gödel\" et \"Godel\" se réfèrent probablement à la même personne.",
+		"find-more": "En trouver davantage...",
+		"method": "Méthode",
+		"key-collision": "collision de clés",
+		"nearest-neighbor": "plus proche voisin",
+		"keying-function": "Fonction de codage",
+		"fingerprint": "empreinte",
+		"ngram": "empreinte N-grammes",
+		"metaphone": "metaphone3",
+		"phonetic": "phonétique de Cologne",
+		"distance-fun": "Fonction distance :",
+		"leven": "Levenshtein",
+		"ppm" : "PPM",
+		"ngram-size": "Taille N-grammes",
+		"ngram-radius": "Rayon",
+		"block-chars": "Bloc de caractères",
+		"reorder-column": "Trier / Supprimer des colonnes",
+		"drag-column": "Glisser des colonnes pour les trier",
+		"drop-column": "Déposer des colonnes ici pour les supprimer",
+		"template-export": "Export du modèle",
+		"template-prefix": "Préfixe",
+		"template-rowt": "Modèle de la ligne",
+		"template-rows": "Séparateur de ligne",
+		"template-suffix": "Suffixe",
+		"idling": "Attente...",
+		"updating": "Mise à jour...",
+		"scatterplot-matrix": "Matrice de nuage de points",
+		"focusing-on": "centre sur",
+		"processing": "Traitement...",
+		"error-getColumnInfo": "Erreur lors de l’appel de 'get-columns-info'",
+		"no-column-dataset": "Cet ensemble de données ne contient aucune colonne.",
+		"linear-plot": "Tracé linéaire",
+		"logarithmic-plot": "Tracé logarithmique",
+		"rotated-counter-clock": "Tourné de -45°",
+		"no-rotation": "Sans rotation",
+		"rotated-clock": "Tourné de 45°",
+		"small-dot": "Point de petite taille",
+		"regular-dot": "Point de taille normale",
+		"big-dot": "Point de grande taille",
+		"cell-fields": "La cellule en cours. Elle a deux champs : 'value' et 'recon'.",
+		"cell-value": "La valeur de la cellule en cours. C’est un alias de 'cell.value'.",
+		"row-fields": "La ligne en cours. Elle a cinq champs : 'flagged', 'starred', 'index', 'cells' et 'record'.",
+		"cells-of-row": "Les cellules de la ligne en cours. C’est un alias de 'row.cells'. Une cellule spécifique peut être retrouvée avec 'cells.<nom de la colonne>' lorsque <nom de la colonne > est un mot simple et avec 'cells.[\"<nom de la colonne>\"]' dans les autres cas.",
+		"row-index": "L’index de la ligne en cours. C’est un alias de 'row.index'.",
+		"returns": "renvoie",
+		"from": "De",
+		"expression": "Expression",
+		"reuse": "Réutiliser",
+		"remove": "Supprimer",
+		"error": "Erreur",
+		"no-syntax-err": "Pas d’erreur de syntaxe",
+		"internal-err": "Erreur interne",
+		"language": "Langue",
+		"preview": "Aperçu",
+		"history": "Historique",
+		"starred": "Étoilée",
```

Comment:
This is in the *edit cells* > *transform windows* expression windows.

![screenshot from 2015-04-21 08 30 59](https://cloud.githubusercontent.com/assets/2589584/7252211/f4cdde62-e800-11e4-9f1a-7447da218fa6.png)